cmd/buildah/delete.go

```diff
@@ -7,41 +7,23 @@ import (
 )
 
 var (
-	deleteFlags = []cli.Flag{
-		cli.StringFlag{
-			Name:  "name",
-			Usage: "`name or ID` of the working container",
-		},
-		cli.StringFlag{
-			Name:   "root",
-			Usage:  "root `directory` of the working container",
-			EnvVar: "BUILDAHROOT",
-		},
-	}
 	deleteDescription = "Deletes a working container, unmounting it if necessary"
 )
 
 func deleteCmd(c *cli.Context) error {
 	args := c.Args()
-	name := ""
-	if c.IsSet("name") {
-		name = c.String("name")
-	}
-	root := c.String("root")
-	if name == "" && root == "" {
-		if len(args) == 0 {
-			return fmt.Errorf("either a container name or --root, or some combination, must be specified")
-		}
-		name = args[0]
-		args = args.Tail()
+	if len(args) == 0 {
+		return fmt.Errorf("container ID must be specified")
 	}
+	name := args[0]
+	args = args.Tail()
```

Comment:
I don't think this args = args.Tail() line is necessary as I'm not seeing a reference to args after this.